src/components/BaseComponents/BaseHeader.tsx

```diff
@@ -36,10 +39,22 @@ const styles = (theme: Theme) =>
       float: 'right',
     },
     account: {
-      color: theme.palette.divider,
+      color: theme.palette.primary.main,
       fontSize: '30px',
       padding: 0,
     },
+    searchBar: {
+      position: 'absolute',
+      width: '100%',
+      padding: '20px',
+      backgroundColor: 'white',
+    },
+    leftTitle: {
+      float: 'left',
```

Comment:
Not sure what the function of this is, but it has no effect when I comment it out.